region/maridia/inner-pink/Crab Shaft.json

```diff
@@ -438,6 +438,27 @@
         "h_canArtificialMorphIBJ"
       ]
     },
+    {
+      "link": [1, 3],
+      "name": "Sciser Ice Clip Door Lock Skip",
+      "notable": true,
+      "requires": [
+        {"ammo": {"type": "Super", "count": 1}},
+        "canTrickyUseFrozenEnemies",
+        {"or": [
+          "Gravity",
+          "HiJump"
```

Comment:
Can it be done with HiJump?  Seems like a snail clip situation.  When trying the puyo clip method, I was pushed through the crab instead of the door.

Reply:
I still can't get it